src/api/app/components/card_component.html.haml

```diff
@@ -0,0 +1,15 @@
+.col-12.col-md-6.col-lg-4
+  .card.mb-3.repository-card
```

Comment:
`.repository-card` should not be here, right? It shouldn't apply to all card components.